src/state/sagas/scenario/FindScenarioById/FindScenarioByIdData.js

```diff
@@ -33,6 +34,18 @@ export function* fetchScenarioByIdData(action) {
     data.parametersValues = ApiUtils.formatParametersFromApi(data.parametersValues);
 
     ScenariosUtils.patchScenarioWithCurrentUserPermissions(data, userEmail, userId, scenariosPermissionsMapping);
+
+    if (!data.security.currentUserPermissions.includes(ACL_PERMISSIONS.SCENARIO.READ)) {
+      const err = new Error(t('commoncomponents.banner.openScenario', "Scenario can't be openned"));
```

Comment:
Same typo as in the translation file: `openned` -> `opened`

Reply:
done